resolwe_bio/processes/workflows/chemut.yml

```diff
@@ -22,15 +22,86 @@
         - label: INDEL_CHR2
           value: indel_chr2
       default: snv
+    - name: genome
+      label: Reference genome
+      type: data:genome:fasta
     - name: parental_strains
       label: Parental strains
       type: list:data:alignment:bam
     - name: mutant_strains
       label: Mutant strains
       type: list:data:alignment:bam
-    - name: genome
-      label: Reference genome
-      type: data:genome:fasta
+    - name: Vc
+      label: Variant Calling
+      group:
+        - name: advanced
+          label: Advanced options
+          type: basic:boolean
+          default: false
+          required: false
+        - name: br_and_ind_ra
+          label: Do variant base recalibration and indel realignment
+          type: basic:boolean
+          default: false
```

Comment:
If the `default` value is set and cannot be "removed" by user (boolean field is either `true` or `false`) the `required` field does not need to be set to `false`, since it is redundant.

Reply:
In the GUI, anything without a `required: false` tag is marked with an asterisk (defaulting to `required: true`). I've found this confusing when running processes with boolean checkboxes in the GUI, because it seems to indicate that the box needs to be checked.

Reply:
OK, good point!